eth/downloader/queue.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 var (
-	blockCacheLimit   = 8192 // Maximum number of blocks to cache before throttling the download
-	maxInFlightStates = 4096 // Maximum number of state downloads to allow concurrently
+	blockCacheLimit   = 16384 // Maximum number of blocks to cache before throttling the download
```

Comment:
Btw, this PR already reduced the number of queued headers from 256K to 32K, so that's a potential 112MB save.